### PR TITLE
Incorporate timer story to why-coop-coep

### PR DESCRIPTION
**Pull request by agektmr**
Updates "Why you need "cross-origin isolated" for powerful features" to incorporate the story about timer resolution.

### CODE REVIEW

File: src/site/content/en/blog/why-coop-coep/index.md [hunk 1]
```diff
@@ -2,20 +2,20 @@
 title: Why you need "cross-origin isolated" for powerful features
 subhead: >
   Learn why cross-origin isolation is needed to use powerful features such as
-  `SharedArrayBuffer`, `performance.measureUserAgentSpecificMemory()`, and the JS Self-Profiling
-  API.
+  `SharedArrayBuffer`, `performance.measureUserAgentSpecificMemory()`, high
+  resolution timer with a better precision and the JS Self-Profiling API.
```
Comment (mihajlija) [inline src/site/content/en/blog/why-coop-coep/index.md:6]:
```suggestion
  resolution timer with better precision and the JS Self-Profiling API.
```

File: src/site/content/en/blog/why-coop-coep/index.md [hunk 2]
```diff
@@ -2,20 +2,20 @@
...
 description: >
   Some web APIs increase the risk of side-channel attacks like Spectre. To
   mitigate that risk, browsers offer an opt-in-based isolated environment called
   cross-origin isolated. Learn why cross-origin isolation is needed to use
   powerful features such as `SharedArrayBuffer`, `performance.measureUserAgentSpecificMemory()`,
-  and the JS Self-Profiling API.
+  high resolution timer with a better precision and the JS Self-Profiling API.
```
Comment (mihajlija) [inline src/site/content/en/blog/why-coop-coep/index.md:12]:
```suggestion
  high resolution timer with better precision and the JS Self-Profiling API.
```

File: src/site/content/en/blog/why-coop-coep/index.md [hunk 3]
```diff
@@ -101,8 +103,10 @@ This is exactly what COOP+COEP is about.
 
 Under a cross-origin isolated state, the requesting site is considered less
 dangerous and this unlocks powerful features such as `SharedArrayBuffer`,
-`performance.measureUserAgentSpecificMemory()` and the JS Self-Profiling API which could otherwise be
-used for Spectre-like attacks. It also prevents modifying `document.domain`.
+`performance.measureUserAgentSpecificMemory()`, [high resolution
+timers](https://www.w3.org/TR/hr-time/) with a better precision and the JS
```
Comment (mihajlija) [inline src/site/content/en/blog/why-coop-coep/index.md:107]:
```suggestion
timers](https://www.w3.org/TR/hr-time/) with better precision and the JS
```

File: src/site/content/en/blog/why-coop-coep/index.md [hunk 4]
```diff
@@ -239,11 +243,13 @@ protect your website in browsers that don't support COOP.
 ## Summary {: #summary }
 
 If you want guaranteed access to powerful features like `SharedArrayBuffer`,
-`performance.measureUserAgentSpecificMemory()` or JS Self-Profiling API, just remember that your
-document needs to use both COEP with the value of `require-corp` and COOP with
-the value of `same-origin`. In the absence of either, the browser will not
-guarantee sufficient isolation to safely enable those powerful features. You can
-determine your page's situation by checking if
+`performance.measureUserAgentSpecificMemory()`, [high resolution
+timers](https://www.w3.org/TR/hr-time/) with a better precision or JS
```
Comment (mihajlija) [inline src/site/content/en/blog/why-coop-coep/index.md:247]:
```suggestion
timers](https://www.w3.org/TR/hr-time/) with better precision or JS
```

**Review by mihajlija — APPROVED** [review]

File: src/site/content/en/blog/why-coop-coep/index.md [hunk 5]
```diff
@@ -86,7 +86,9 @@ successful.
 This all changed with
 [Spectre](https://en.wikipedia.org/wiki/Spectre_(security_vulnerability)), which
 makes any data that is loaded to the same browsing context group as your code
-potentially readable. If `evil.com` embeds a cross-origin image, they can use a
+potentially readable. By measuring the time certain operations take, attackers
+can guess the contents of the CPU caches, and through that, the contents of the
+process' memory. If `evil.com` embeds a cross-origin image, they can use a
```
Comment (yoavweiss) [inline src/site/content/en/blog/why-coop-coep/index.md:91]: I'd add here: "Such timing attacks are possible with low-granularity timers that exist in the platform, but can be sped up with high-granularity timers, both explicit (like `performance.now()`) and implicit (like SharedArrayBuffers)"

Comment (agektmr) [reply src/site/content/en/blog/why-coop-coep/index.md]: Thanks for the suggestion. Added.

**Review by yoavweiss — APPROVED** [review]
LGTM

**Pull request merged**